platform/mellanox/mlnx-platform-api/sonic_platform/chassis.py

```diff
@@ -387,26 +387,30 @@ def get_change_event(self, timeout=0):
             self.sfp_event.initialize()
 
         wait_for_ever = (timeout == 0)
+        # select timeout should be no more than 1000ms to ensure fast shutdown flow
```

Comment:
@Junchao-Mellanox may be add more context in the comment why max timeout is kept 1sec?